vscode/src/azure/networkRequests.ts

```diff
@@ -27,6 +30,16 @@ export async function azureRequest(
 
     if (!response.ok) {
       log.error("Azure request failed", response);
+      if (associationId) {
```

Comment:
Do you happen to know if there is some kind of correlation ID that we can send _to_ the service that will help them correlate our events with theirs?